src/banknames.json

```diff
@@ -293,7 +293,8 @@
   "DLXB": "Dhanlaxmi Bank",
   "DMCB": "Deccan Merchants Co-operative Bank",
   "DMCX": "Durgapur Mahila Co-operative Bank",
-  "DMKB": "Dmk Jaoli Bank",
+  "DMKB": "Dattatraya Maharaj Kalambe Jaoli Sahakari Bank",
```

Comment:
DMKB has `Dattatraya Maharaj Kalambe Jaoli Sahakari Bank` as per csv, the same bank also has a sublet with ICICI